### ISSUE TITLE
[BUG] Plots, Farm,and Pool Feature wont update or load after update on windows

### ISSUE THREAD

**Issue by cdawg2020** [Chia-Network/chia-blockchain#7391]
Plots, Farm,and Pool Feature wont update or load after update on windows was runnung 308 Solo plots..

**Expected behavior**

**Screenshots**

**Desktop**

- OS: <!-windows-->
- OS Version/Flavor: 
- CPU: 

**Additional context**

**Comment by loppefaaret** [comment]
try these 3 steps, checking inbetweem if one worked for you:
WINDOWS COMPLETE CONFIG RESET

Close down the Chia app, and open up a PowerShell window, and execute the below commands one at a time:
```
mv ~\.chia\mainnet ~\.chia\old
cd ~\AppData\Local\chia-blockchain\app-1.2.0\resources\app.asar.unpacked\daemon\
.\chia.exe init
mv ~\.chia\old\db ~\.chia\mainnet\
mv ~\.chia\old\wallet ~\.chia\mainnet\
```

WINDOWS WALLET DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
mv ~\.chia\mainnet\wallet\db\block* ~\Desktop\
```
Then try opening up the Chia app again
if it worked, you can remove the files we put on the Desktop!

WINDOWS BLOCKCHAIN DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
mv ~\.chia\mainnet\db\block* ~\Desktop\
```
Then try opening up the Chia app again
if it worked, you can remove the files we put on the Desktop!

**Comment by Sp4rkR4t** [comment]
![image](https://user-images.githubusercontent.com/3926906/125299012-a0226780-e320-11eb-8620-9cd0fd267727.png)

I'm having the same issue on windows so tried these steps and I get hit with a permission denied message when I put in the first line.
Instead I backed up the wallet & db to a new location then did the init and copied them back, upon restarting chia it had lost all my plot folders and my pool info.

I moved onto the next step `mv ~\.chia\mainnet\wallet\db\block* ~\Desktop\` and the folders and pool was still gone.

I then tried final step `mv ~\.chia\mainnet\db\block* ~\Desktop\` and while it is syncing from the start again both my plot directories and my pool info are gone.
